package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ml-frontend",
   "scripts": {
-    "test:cli": "turbo run build --filter=start-frontend test --filter=start-frontend",
+    "test:cli": "turbo run build --filter=start-frontend && turbo run test --filter=start-frontend",
```

Comment:
I'm still not understanding why it passes locally, but not on the CI. 🤔 
Do you think updating to the most recent turbo version would change the outcome?

Reply:
Same as Dan-san.
I don't get why just split test part explicitly pass the cli test remote.

Like I said, I see the build part is successfully and generate directory under temp, however, it contains some extra directory and I suppose that is generated by github.
That's the difference between local and remote and I assume this is why test is failing in remote.

Test is passed in local because there is no directory added on local, however, test is failed on remote because somehow extra directory(_github_workflow, _runner_file_commands etc...) added on generated folder.

I modified test case, then it works fine, and also revert back the test to original one, then fail again.

Test modify commit
https://github.com/monstar-lab-oss/ml-frontend/pull/1261/commits/386bbdb4b9493ea1621ced16a1259759069fb8cf

Test result after modify(passed)
https://github.com/monstar-lab-oss/ml-frontend/actions/runs/9967221235/job/27540555803

change back test to original
https://github.com/monstar-lab-oss/ml-frontend/pull/1261/commits/4e1a548a05d07b72495d4dc1da7db23e24dacad0

Test result revert back to the original(failed)
https://github.com/monstar-lab-oss/ml-frontend/actions/runs/9967285369/job/27540720792

Reply:
@jinmayamashita cc: @ptrkdan 

Thank you for discuss about it, I understand split test still generate folder properly(still not sure why they take out the extra folders that generates by github tho), and passed test.

I'll approve it.